pkg/webhooks/ingresscontroller/ingresscontroller.go

```diff
@@ -149,12 +176,151 @@ func (wh *IngressControllerWebhook) authorized(request admissionctl.Request) adm
 		}
 	}
 
-	ret = admissionctl.Allowed("IngressController operation is allowed")
+	/* TODO:
+	 * 1) ONLY check for privatelink clusters? How?
+	 * 2) HCP vs Classic, etc.. ...any other cluster type this should apply to? Is this handled by the
+	 * HypershiftEnabled vs ClassicEnabled flags set in this module? Currently HCP disabled.
+	 * If HCP is to be enabled for allowed source ranges, should this part of a 2nd ingress validator to
+	 * allow separation of validations between cluster install types? Is there a run time method available
+	 * to this validator to determine classic vs hcp?
+	 * 3) What other specifics should be checked here for this cidr check to be applicable?m
```

Comment:
AFAIK, Machine CIDR is the _only_ check that would have an impact on this, so no other checks should be necessary

Reply:
Thanks. This comment/concern has been removed.